src/Application/Games/Commands/GetGameUserCommand.cs

```diff
@@ -266,7 +266,6 @@ private async Task<IList<EquippedItem>> GiveUserRandomItemSet(User user)
                     {
                         ItemId = item.Id,
                         User = user,
-                        Rank = 0,

```

Comment:
OPT: would be nice to explicitly set all properties (i.e. Is broken)

Reply:
done